.github/workflows/generate-summary.yaml

```diff
@@ -429,7 +429,7 @@ jobs:
           printf 'A list of all unresolved "internal compiler error", "Segmentation fault", "test for excess errors" failures present at this hash\n\n' >> trimmed_comment.md
           printf -- '---\n\n' >> trimmed_comment.md
           head -c 65000 unresolved_important_failures.md >> trimmed_comment.md
-          if [ $(cat trimmed_comment.md | wc -l) -ne $(cat unresolved_important_failures.md | wc -l) ]; then printf "\n\`\`\`\nComment text has been trimmed. Check artifact logs for full list.\n" >> trimmed_comment.md; fi
+          if [ "$(stat -c%s unresolved_important_failures.md)" -gt 65000 ]; then printf "\n\`\`\`\nComment text has been trimmed. Check artifact logs for full list.\n" >> trimmed_comment.md; fi
```

Comment:
Could you also remove the extra `\`\`\`\n` as well then?

Reply:
Yeah. If you take a look at the end of unresolved important failures comments, you can see that the "Comment has been trimmed" message is inside of a code block. Removing the \`\`\` before the "Comment has been trimmed" should fix it